src/main/kotlin/org/rust/lang/core/resolve2/FacadeResolve.kt

```diff
@@ -302,24 +302,24 @@ private fun getMacroIndexInParent(item: PsiElement, parent: PsiElement, crate: C
     return if (itemStub != null && parentStub != null) {
         parentStub.childrenStubs.asSequence()
             .takeWhile { it !== itemStub }
-            .count { it.hasMacroIndex(crate) }
+            .count { it.hasMacroIndex() }
     } else {
         parent.children.asSequence()
             .takeWhile { it !== item }
-            .count { it.hasMacroIndex(crate) }
+            .count { it.hasMacroIndex() }
     }
 }
 
 fun PsiElement.findItemWithMacroIndex(macroIndexInParent: Int, crate: Crate): PsiElement {
```

Comment:
`crate` parameter is unused here and in `getMacroIndexInParent`. Actually I believe we can even remove `crate` from `RsModInfo` now, but probably it is for another PR